public/app/plugins/panel/timeseries/config.ts

```diff
@@ -177,14 +179,19 @@ export function getGraphFieldConfig(cfg: GraphFieldConfig): SetFieldConfigOption
       commonOptionsBuilder.addAxisConfig(builder, cfg);
       commonOptionsBuilder.addHideFrom(builder);
 
-      builder.addSelect({
-        path: 'thresholdsStyle.mode',
```

Comment:
We use nested paths in other places, what is the problem in this case?

Reply:
see my comment below